openapi/components/schemas/assessment-rubric-feedback.yaml

```diff
@@ -0,0 +1,60 @@
+title: An Assessment Rubric item
+description: An Assessment Rubric item
+readOnly: true
+type: object
+properties:
+  capabilities:
```

Comment:
May I have some descriptions of these properties? The rubric documentation looks quite barren:
![image](https://user-images.githubusercontent.com/407053/196590513-860c233c-6b2f-4ce7-93af-7abc91b57964.png)

Reply:
updated descriptions
![image](https://user-images.githubusercontent.com/4447143/196596786-1ba183c6-1e4d-4ffd-8ac0-87715c5a2587.png)